src/com/google/javascript/jscomp/CoverageInstrumentationPass.java

```diff
@@ -78,6 +78,13 @@ public void process(Node externsNode, Node rootNode) {
             compiler,
             rootNode,
             new BranchCoverageInstrumentationCallback(compiler, instrumentationData));
+      } else if (instrumentOption == InstrumentOption.PRODUCTION) {
+        NodeTraversal.traverse(
+            compiler,
+            rootNode,
+            new ProductionCoverageInstrumentationCallback(compiler));
+
+        return;
```

Comment:
Add comment explaining that we don't need to add header code for production coverage because it relies on instrument function being part of source code.